datadog_checks_base/datadog_checks/base/data/agent_requirements.in

```diff
@@ -99,17 +99,17 @@ rethinkdb==2.4.9
 scandir==1.10.0
 securesystemslib[crypto,pynacl]==0.28.0; python_version > '3.0'
 semver==2.13.0; python_version < '3.0'
-semver==3.0.1; python_version > '3.0'
+semver==3.0.2; python_version > '3.0'
 serpent==1.41; sys_platform == 'win32' and python_version > '3.0'
 service-identity[idna]==21.1.0
-simplejson==3.19.1
+simplejson==3.19.2
 six==1.16.0
-snowflake-connector-python==3.1.0; python_version > '3.0'
+snowflake-connector-python==3.4.0; python_version > '3.0'
```

Comment:
Could you also bump it here? https://github.com/DataDog/datadog-agent/blob/302cdc8a846bfd08da9244706545b3f22d7b34d8/omnibus/config/software/snowflake-connector-python-py3.rb#L2 Otherwise it will have no effect

Reply:
Thanks!
https://github.com/DataDog/datadog-agent/pull/20631

Which PR needs to be merged first? 🐔 🥚

Reply:
Both at the same time 😅